q2_minimap2/_filtering_utils.py

```diff
@@ -106,22 +106,23 @@ def process_sam_file(input_sam_file, keep_mapped, min_per_identity):
             # Logic for including or excluding reads based on mappings and
             # identity percentage
             if keep_mapped:
-                # Includes reads that are mapped and not secondary, based on the
```

Comment:
Hey @ChristosMatzoros just a small question here. The flag `keep_mapped` essentially decides if the output contains the mapped or the unmapped sequences.

Reply:
Yes exactly, based on the parameter that the user has provided it decides if it will keep either the mapped or the unmapped sequences.